package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "webknossos",
-  "version": "0.17.146",
+  "version": "1.10.1",
```

Comment:
I think this change was added when you executed `yarn version`. Can you undo this?